src/frontend/src/eth/components/fee/FeeDisplay.svelte

```diff
@@ -11,7 +11,7 @@
 
 	const { feeStore: feeData, feeSymbolStore }: FeeContext = getContext<FeeContext>(FEE_CONTEXT_KEY);
 
-	let fee: BigNumber | undefined | null = undefined;
+	export let fee: BigNumber | undefined | null = undefined;
```

Comment:
Can you add a TODO, noticing it now, I don't think we should solve this by exposing the `fee`. That's a shortcut.

I should have noticed it this morning, sorry.

I'll take care of it next week.